bears/general/AnnotationBear.py

```diff
@@ -9,21 +9,22 @@
 
 class AnnotationBear(LocalBear):
 
-    def run(self, filename, file, language: str):
+    def run(self, filename, file, language: str, coalang_dir: str = None):
```

Comment:
does this commit even fit into the PR "Add Intendation bear"?

It's not a problem since in the end its all about the commit but if this is independent we can probably just merge it in an own PR.